sirepo/pkcli/jupyterhublogin.py

```diff
@@ -58,9 +60,21 @@ def maybe_create_sirepo_user(qcall, email, display_name):
         with qcall.auth.logged_in_user_set(u, method=sirepo.auth.METHOD_EMAIL):
             n = sirepo.sim_api.jupyterhublogin.create_user(
                 qcall=qcall,
-                check_dir=True,
             )
             qcall.auth_db.model("UserRole").add_roles(
                 roles=[sirepo.auth_role.for_sim_type("jupyterhublogin")],
             )
         return PKDict(email=email, jupyterhub_user_name=n)
+
+
+def old_user_dirs():
```

Comment:
After looking at this I think something a bit more generic would be good. Call it "uknown_user_dirs" and just output the dirs on a new line but no `rm -rf`. It could be useful in the future. And it is easy enough to add the `rm -rf` from the outside.

Reply:
[ad6298a](https://github.com/radiasoft/sirepo/pull/7027/commits/ad6298ae14d580069dd27cda09abc4bfad79f545)